src/main/java/woowacourse/auth/ui/AuthenticationInterceptor.java

```diff
@@ -16,6 +17,10 @@ public AuthenticationInterceptor(JwtTokenProvider jwtTokenProvider) {
 
     @Override
     public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
+        if (HttpMethod.OPTIONS.matches(request.getMethod())) {
+            return true;
+        }
+
```

Comment:
이 부분으로 CORS 에러를 처리하셨군요.
이 부분은 궁금한 점인데 preflight 요청에는 AUTHORIZATION 헤더가 담기지 않나요? OPTIONS일 경우 인터셉터 통과를 시키는 처리를 했을 때 CORS 에러가 해결되는 이유가 궁금합니다 :)

Reply:
글 정리해왔습니다!

Reply:
기다렸던 글이네요! 잘 정리해주셨습니다 👍 
조금만 더 깊게 다뤄보면
> For a [CORS-preflight request](https://fetch.spec.whatwg.org/#cors-preflight-request), [request](https://fetch.spec.whatwg.org/#concept-request)’s [credentials mode](https://fetch.spec.whatwg.org/#concept-request-credentials-mode) is always "same-origin", i.e., it excludes credentials, but for any subsequent [CORS requests](https://fetch.spec.whatwg.org/#cors-request) it might not be. Support therefore needs to be indicated as part of the HTTP response to the [CORS-preflight request](https://fetch.spec.whatwg.org/#cors-preflight-request) as well. [출처](https://fetch.spec.whatwg.org/#http-responses)

preflight 요청에는 Auth 헤더, 쿠키, tls 인증 등등 각종 credential 들이 빠집니다. 나중에 CORS 관련 에러를 마주치면 꼭 지금을 떠올리시길 바랍니다 :)

+) 스프링은 CorsUtils.isCorsRequest() 라는 유틸을 제공합니다. 활용해보시면 좋을 것 같아요.